hudi-common/src/main/java/org/apache/hudi/common/bloom/HoodieDynamicBoundedBloomFilter.java

```diff
@@ -125,5 +126,42 @@ private void extractAndSetInternalBloomFilter(DataInputStream dis) throws IOExce
     internalDynamicBloomFilter = new InternalDynamicBloomFilter();
     internalDynamicBloomFilter.readFields(dis);
   }
+
+  @Override
+  public void or(BloomFilter other) {
```

Comment:
unit tests

Reply:
In TestHoodieFileMetadataMerger#testBloomFilter

Reply:
Same. please move them to the UT for this class. I know it's a bit scattered now . but change starts somewhere :)